.github/workflows/tests.yaml

```
@@ -0,0 +1,31 @@
+name: Run tests
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: ['ubuntu-18.04', 'ubuntu-latest', 'macos-11', 'windows-2016']
```

Is there a reason you test on 2 ubuntu versions and on specific older macos and windows versions? Would the following also work?
```suggestion
        os: [ubuntu-latest, macos-latest, windows-latest]
```

---

@lukpueh I just put random versions in to test compat 😄 you can change this to any valid OS